plugins/modules/aws_kms.py

```diff
@@ -821,18 +821,19 @@ def create_key(connection, module):
     if module.params.get('policy'):
         params['Policy'] = module.params['policy']
 
+    if module.check_mode:
+        return {'changed': True}
+
     try:
         result = connection.create_key(**params)['KeyMetadata']
     except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError) as e:
         module.fail_json_aws(e, msg="Failed to create initial key")
     key = get_key_details(connection, module, result['KeyId'])
-
```

Comment:
I don't think this blank line should be removed.

Reply:
Inadvertent when I made the restruct change. Corrected.